### ISSUE TITLE
The current way of posting an applet here is not very practical. 

### ISSUE THREAD

**Issue by raphaelquintao** [linuxmint/cinnamon-spices-applets#4252]
Hi guys.
I'm here to point some troubles I'm having as author a maintainer of a applet here  [QRedshift](https://cinnamon-spices.linuxmint.com/applets/view/313).

Basically this repository holds all the applets itself, so peoples make a pull request here and on the original repository on my account.
The problem is: i need to keep it synced, so from time to time i need manually see whats has changes here and update my original repository and push it back here. That's not very practical and really boring.

My suggestion is to make something more "intelligent" like the [Sublime Package Control](https://github.com/wbond/package_control_channel) does.
Basically the repository mentioned above is a index to the authors repository, and uses the GitHub releases to control the versions of our apps. Its works a lot better.

Notify: @brownsr

**Comment by Gr3q** [comment]
@brownsr will probably say that their use case for using a monorepo is not about controlling versions but code (meaning it's reviewed by someone from the Cinnamon team)

I don't know what other options there are while maintaining code control from their end so I can't really offer any alternatives. 

Still, I personally don't have a problem keeping applet code in this repo, although I have problems how applets are handled that have some kind of build process implemented (but that's a topic for another thread).

**Comment by fredcw** [comment]
One good thing about having all the applets in one repository is it easy to search them to find out how they are using the cinnamon API.
